config/test_cases/test_branch_origin_integration.yml

```diff
@@ -6,4 +6,4 @@ extensions:
       title: "run integration tests"
       repository: "origin"
       script: |-
-        JUNIT_REPORT='true' make test -o check -k
\ No newline at end of file
+        OS_BUILD_ENV_PRESERVE=_output/local hack/env JUNIT_REPORT='true' make test -o check -k
```

Comment:
Does this work? I would not expect `test/end-to-end.sh` to work inside `hack/env`. No `docker` client there.

Also you'll want to mount `/tmp:/openshifttmp` like in the `test_branch_origin_check` so we can grab artifacts back from the container.